caddyhttp/maxrequestbody/maxrequestbody_test.go

```diff
@@ -0,0 +1,118 @@
+package maxrequestbody
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/mholt/caddy/caddyhttp/httpserver"
+)
+
+const (
+	KB = 1024
+	MB = 1024 * 1024
+	GB = 1024 * 1024 * 1024
+)
+
+func TestParseArguments(t *testing.T) {
+	cases := []struct {
+		arguments []string
+		expected  []httpserver.PathLimit
+		hasError  bool
+	}{
+		// Parse errors
+		{arguments: []string{"123.5"}, expected: []httpserver.PathLimit{}, hasError: true},
+		{arguments: []string{"200LB"}, expected: []httpserver.PathLimit{}, hasError: true},
+		{arguments: []string{"path:999MB"}, expected: []httpserver.PathLimit{}, hasError: true},
+		{arguments: []string{"1_234_567"}, expected: []httpserver.PathLimit{}, hasError: true},
+
+		// Valid results
+		{arguments: []string{}, expected: []httpserver.PathLimit{}, hasError: false},
+		{
+			arguments: []string{"100"},
+			expected:  []httpserver.PathLimit{httpserver.PathLimit{Path: "/", Limit: 100}},
+			hasError:  false,
+		},
+		{
+			arguments: []string{"100KB"},
+			expected:  []httpserver.PathLimit{httpserver.PathLimit{Path: "/", Limit: 100 * KB}},
+			hasError:  false,
+		},
+		{
+			arguments: []string{"100MB"},
+			expected:  []httpserver.PathLimit{httpserver.PathLimit{Path: "/", Limit: 100 * MB}},
+			hasError:  false,
+		},
+		{
+			arguments: []string{"100GB"},
+			expected:  []httpserver.PathLimit{httpserver.PathLimit{Path: "/", Limit: 100 * GB}},
+			hasError:  false,
+		},
+		{
+			arguments: []string{"index", "100"},
+			expected:  []httpserver.PathLimit{httpserver.PathLimit{Path: "/index", Limit: 100}},
+			hasError:  false,
+		},
+		{
+			arguments: []string{"/home", "100MB", "/upload/images", "500GB"},
+			expected: []httpserver.PathLimit{
+				httpserver.PathLimit{Path: "/home", Limit: 100 * MB},
+				httpserver.PathLimit{Path: "/upload/images", Limit: 500 * GB},
+			},
+			hasError: false},
+		{
+			arguments: []string{"999", "/home", "12345MB"},
+			expected: []httpserver.PathLimit{
+				httpserver.PathLimit{Path: "/", Limit: 999},
+				httpserver.PathLimit{Path: "/home", Limit: 12345 * MB},
+			},
+			hasError: false,
+		},
```

Comment:
Oh gofmt wasn't the reason. You guys are using simplify mode (-s).

I'll update this.